code/implementation/docker/machine/pom.xml

```diff
@@ -39,5 +39,10 @@
             <scope>provided</scope>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
What code changed in the docker/machine project that needed this?  The github diff doesn't show me full filenames